evm/db/chain.py

```diff
@@ -244,9 +247,10 @@ def commit(self, checkpoint):
     def clear(self):
         self.db.clear()
 
-    def get_state_db(self, state_root, read_only, access_list=None):
+    def get_state_db(self, state_root, read_only, access_list=None, is_stateless=False):
+        db = TrackedDB(self.db) if is_stateless else self.db
```

Comment:
Same thought here about avoiding inline turnary statements in favor of readability.